test/rhsmlib/test_file_monitor.py

```diff
@@ -35,6 +39,11 @@ def setUp(self):
         self.fsw1 = file_monitor.FilesystemWatcher(self.dir_list)
         self.fsw2 = file_monitor.InotifyFilesystemWatcher(self.dir_list)
 
+        now: float = time.time()
+        self.future = now + 2.0
```

Comment:
hm maybe 2 seconds is not much, in case the tests take more than that to run -- maybe bump it to 60 or even 120?